Wit/RecordingSessionStarter.h

```diff
@@ -1,17 +0,0 @@
-//
-//  RecordingSessionStarter.h
-//  Wit
-//
-//  Created by Aric Lasry on 8/14/14.
-//  Copyright (c) 2014 Willy Blandin. All rights reserved.
-//
-
-#import <Foundation/Foundation.h>
-#import "WITRecorder.h"
-
-@protocol RecordingSessionStarter <NSObject>
-
--(void)sessionDidStart:(WITRecorder *)recorder;
--(void)sessionEnded:(WITRecorder *)recorder;
-
-@end
```

Comment:
Why did we delete this file? Does this fit under "Fix some retain cycles" or  "improve code organization and style'?